cerberus/devops/cerberus-sts.yml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cerberus-server
+  namespace: openshift-bcgov-cerberus
+  labels:
+    app: cerberus
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      tool: Cerberus
+  template:
+    metadata:
+      labels:
+        tool: Cerberus
+    spec:
+      serviceAccountName: cerberus
+      imagePullSecrets:
+        - name: artifacts-platsvcs-reader
+      containers:
+        - name: cerberus
+          imagePullPolicy: Always
+          # NOTE: Dockerfile changes filesystem permission for monitoring output storage. otherwise we'll need to use privileged account
+          # securityContext:
+          #   privileged: true
+          image: artifacts.developer.gov.bc.ca/plat-util-images/cerberus:test
+          command: ["/bin/sh", "-c"]
+          args: ["/usr/local/bin/entrypoint.sh"]
+          ports:
+            - containerPort: 8080
+          resources:
+            limits:
+              cpu: '1'
+              memory: 1Gi
+            requests:
+              cpu: 100m
+              memory: 256Mi
+          volumeMounts:
+            - mountPath: "/tmp/kubeconfig-template"
+              subPath: kubeconfig-template
+              name: kubeconfig
+            - mountPath: "/tmp/cerberus-config-template.yaml"
+              subPath: cerberus-config-template.yaml
+              name: cerberus-config
+            - mountPath: /tmp/data
```

Comment:
We don't mound custom check anymore?

Reply:
no, they are added in as part of the image build

Reply:
nvm, I saw the docker file...